addons/account/tests/test_account_payment_register.py

```diff
@@ -320,12 +320,12 @@ def test_register_payment_single_batch_grouped_writeoff_higher_amount_credit(sel
             'payment_difference_handling': 'reconcile',
             'writeoff_account_id': self.company_data['default_account_revenue'].id,
             'writeoff_label': 'writeoff',
-            'payment_method_line_id': self.inbound_payment_method_line.id,
+            'payment_method_line_id': self.outbound_payment_method_line.id,
         })._create_payments()
 
         self.assertRecordValues(payments, [{
             'ref': 'BILL/2017/01/0001 BILL/2017/01/0002',
-            'payment_method_line_id': self.inbound_payment_method_line.id,
```

Comment:
if it's a vendor bill, shouldn't it be an inbound payment method? :thinking:

Reply:
I don't think so, you have to pay your bill, so the payment is 'outbound' and so the payment method